packages/web3-utils/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "web3-utils",
+	"version": "4.0.0-alpha.0",
+	"description": "Collection of utility functions used in web3.js.",
+	"main": "dist/index.js",
+	"repository": "https://github.com/ChainSafe/web3.js",
+	"author": "ChainSafe Systems",
+	"license": "LGPL-3.0",
+	"files": [
+		"dist/**/*"
+	],
+	"scripts": {
+		"clean": "rimraf dist || mkdir dist || true",
```

Comment:
This should've been mentioned in previous PRs, but:

- Why are we using `||` instead of `&&`
- Why is making the dir apart of cleaning
- What's the `true` for?

Reply:
- If the command on left hand failed means directory does not exists to clean then only right side will execute an create the directory 
- Because cleaning refers to the remove all the files in the directory not the directory itself. 
- The last true make sure that whole script still proceed to next step in case of any foreseen error.

Reply:
I don't think an empty `dist` directory should be hanging around. The build command can handle making the directory if it doesn't exist. Additionally, I don't think this command has to execute successfully, otherwise we're suppressing any potential errors